### PR TITLE
friends list styling links working

### PR DESCRIPTION
**Pull request by Emmah-lou**
have a look, basic styling done also set up nav links to work properly

### CODE REVIEW

**Review by BlocksOnChainCode — COMMENTED** [review]
I'm deff all for reviewing this way, keeps everything more clean, commit messages, branches etc.

File: src/components/layout/FriendsList.scss
[the remark this answers is not in the document]
Comment (Emmah-lou) [reply src/components/layout/FriendsList.scss:1]: what line?

File: src/components/layout/FriendsList.jsx [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
-        })}
-    </ul>
+    <>
+      <h2 className="friends-list__title ml-6">Friends List</h2>
+      <section className="friends-list">
+        {friends &&
+          friends.map((friend) => {
+            return <Friend key={friend.id} friend={friend} />;
+          })}
+      </section>
```
[the remark this answers is not in the document]
Comment (Emmah-lou) [reply src/components/layout/FriendsList.jsx:16]: yea i think my brain just said PUT A DIV

couldnt fight off the intrusive thoughts lol jk

File: src/components/layout/FriendsList.scss
[the remark this answers is not in the document]
Comment (Emmah-lou) [reply src/components/layout/FriendsList.scss:1]: nvm see the syntax..... BLONDE!

**Pull request merged**